### PR TITLE
[Chat] Version bump of communication-chat

### PR DESCRIPTION
**Pull request by JoshuaLai**
# What
Version bump communication-chat to 1.6.0-beta2

# Why
Introducing the policy violation property 

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by palatter — APPROVED** [review]
Looks good, can you add changelog that details removal of api? Thanks!

**Review by jimchou-dev — APPROVED** [review]

File: packages/chat-stateful-client/package.json [hunk 1]
```diff
@@ -38,11 +38,11 @@
...
   },
   "peerDependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || >=1.5.0"
+    "@azure/communication-chat": "1.6.0-beta.2 || >=1.5.0"
   },
   "devDependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || ^1.5.0",
-    "@azure/communication-signaling": "1.0.0-beta.23 || 1.0.0-beta.26",
+    "@azure/communication-chat": "1.6.0-beta.2 || ^1.5.0",
+    "@azure/communication-signaling": "1.0.0-beta.27 || 1.0.0-beta.26",
```
Comment (emlynmac) [inline packages/chat-stateful-client/package.json:45]: Can we remove this signaling dependency as it should be part of the chat deps?

Also, why are these dependencies under devDependencies?

File: packages/fake-backends/package.json [hunk 2]
```diff
@@ -29,9 +29,9 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || ^1.5.0",
+    "@azure/communication-chat": "1.6.0-beta.2 || ^1.5.0",
     "@azure/communication-common": "^2.3.1",
-    "@azure/communication-signaling": "1.0.0-beta.23 || 1.0.0-beta.26",
+    "@azure/communication-signaling": "1.0.0-beta.27 || 1.0.0-beta.26",
```
Comment (emlynmac) [inline packages/fake-backends/package.json:34]: Does this need to be here still?

File: packages/react-composites/package.json [hunk 3]
```diff
@@ -82,9 +82,9 @@
   "devDependencies": {
     "@azure/communication-calling": "1.26.1-beta.1 || ^1.25.1",
     "@azure/communication-calling-effects": "1.0.1",
-    "@azure/communication-chat": "1.6.0-beta.1 || ^1.5.0",
+    "@azure/communication-chat": "1.6.0-beta.2 || ^1.5.0",
```
Comment (emlynmac) [inline packages/react-composites/package.json:85]: Why is this both a dev and peer dependency?

File: packages/react-composites/package.json [hunk 4]
```diff
@@ -82,9 +82,9 @@
...
     "@azure/communication-identity": "^1.3.0",
-    "@azure/communication-signaling": "1.0.0-beta.23 || 1.0.0-beta.26",
+    "@azure/communication-signaling": "1.0.0-beta.27 || 1.0.0-beta.26",
```
Comment (emlynmac) [inline packages/react-composites/package.json:87]: Can we remove this, as contained within chat sdk?

File: packages/chat-stateful-client/package.json (hunk 1 above)
Comment (JoshuaLai) [reply packages/chat-stateful-client/package.json:45]: We directly call to `"@azure/communication-signaling"` in the chat stateful test so probably why they are just in dev dependencies.

File: packages/chat-component-bindings/package.json [hunk 5]
```diff
@@ -38,12 +38,12 @@
...
   },
   "peerDependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || >=1.5.0",
+    "@azure/communication-chat": "1.6.0-beta.2 || >=1.5.0",
     "@types/react": ">=16.8.0 <19.0.0",
     "react": ">=16.8.0 <19.0.0"
   },
   "devDependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || ^1.5.0",
+    "@azure/communication-chat": "1.6.0-beta.2 || ^1.5.0",
```
Comment (emlynmac) [inline packages/chat-component-bindings/package.json:46]: should this be here?

File: packages/chat-stateful-client/package.json (hunk 1 above)
Comment (emlynmac) [reply packages/chat-stateful-client/package.json:45]: The chat dependency can go though, right?

**Review by emlynmac — APPROVED** [review]

**Pull request merged**